config.m4

```diff
@@ -1049,7 +1049,9 @@ if test "$PHP_GRPC" != "no"; then
     , $ext_shared, , -fvisibility=hidden \
     -DOPENSSL_NO_ASM -D_GNU_SOURCE -DWIN32_LEAN_AND_MEAN \
     -D_HAS_EXCEPTIONS=0 -DNOMINMAX -DGRPC_ARES=0 \
-    -DGRPC_POSIX_FORK_ALLOW_PTHREAD_ATFORK=1)
+    -DGRPC_POSIX_FORK_ALLOW_PTHREAD_ATFORK=1 \
+    -DGRPC_XDS_USER_AGENT_SUFFIX="\"PHP\"" \
```

Comment:
I think this needs to actually be GRPC_XDS_USER_AGENT_NAME_SUFFIX

This typo may be related to https://github.com/grpc/grpc/issues/26496